client/src/components/dom/Auth/AuthModal/index.ts

```diff
@@ -0,0 +1,3 @@
+import AuthModal from "./AuthModal";
+
+export default AuthModal;
```

Comment:
In general, prefer named exports over default ones. Named exports give us autocompletion in the editor and disallow us from importing things with the wrong name (you can import a default export with any name, which is misleading). You can always rename your imports explicitly, if needed:

```ts
import { MyComponent as YourComponent } from 'myComponent';

return (
	<YourComponent />
)
```